### ISSUE TITLE
Add Shapefile automatically to map

### ISSUE THREAD

**Issue by fzadrazil** [hslayers/hslayers-ng#702]
When adding SHP, the layer is stored in Layman, but is not added automatically to the map.

**Comment by DailisLangovskis** [comment]
@jmacura Is this issue still unresolved?

**Comment by jmacura** [comment]
@DailisLangovskis I have checked in both https://hub.lesprojekt.cz/hsl-ng/hsl-ng-1.x/apps/full/ and https://hub.lesprojekt.cz/hsl-ng/hsl-ng-2.x/apps/full/ and obviously not.

**Comment by fzadrazil** [comment]
I confirm this is still an issue. And probably the last one blocking the 2.5 release?

**Comment by raitisbe** [comment]
For me it also worked, but not when trying to add layer title which already existed. The error panel in the UI didnt specify the reason though, which i tried to solve in the commit

**Comment by jmacura** [comment]
I believe the issue is in the Layman re-naming of layer names, once again.
